Parser/pegen.c

```diff
@@ -145,15 +145,12 @@ byte_offset_to_character_offset(PyObject *line, int col_offset)
     if (!str) {
         return 0;
     }
+    assert(col_offset <= strlen(str));
```

Comment:
Nice GH actions integration here. Compiler complains about comparison:
``comparison between signed and unsigned integer expressions [-Wsign-compare]``

Reply:
Fixed in 4654706